consumer_group.go

```diff
@@ -624,6 +627,10 @@ func (s *consumerGroupSession) MarkOffset(topic string, partition int32, offset
 	}
 }
 
+func (s *consumerGroupSession) Commit() {
+	s.offsets.Commit()
+}
```

Comment:
Do we need a similar check of the session ctx here, like we have in consume to prevent us from attempting to commit after a rebalance has already started?

```go
// quick exit if rebalance is due
select {
case <-s.ctx.Done():
	return
case <-s.parent.closed:
	return
default:
}
```

Reply:
@dnwe , that would introduce new behaviour. 
```
        om.flushToBroker()
	om.releasePOMs(false)
```
Above statements were used on the `case <-om.ticker.C` case in the `(om *offsetManager) mainLoop()` function.
Now both (async with 'mainLoop' and sync with 'Commit') use the new 'Commit' function using the exact same contents as above.
```
func (om *offsetManager) Commit() {
	om.flushToBroker()
	om.releasePOMs(false)
}
```